app/components/FlowLayout.js

```diff
@@ -31,71 +27,71 @@ import {connect} from 'react-redux'
 //                         	onPress={()=> {this.props.onDeleteClick()}} />
 // 					</View>
 //                 </TouchableOpacity>
-            
+
 // 			</View>
 // 		);
 // 	};
 
 // }
 class FlowLayout extends Component {
-	
+
 	constructor(props) {
 		super(props);
-    }
+	}
 
 	render() {
 		// console.log('FLOW LAYOUT BCV REF ',this.props.bcvRef)
 		let items = this.props.dataValue.verses.map((value, position) => {
-				return(
-					<View key={position}>
-					<TouchableOpacity onPress={()=>{
-					this.props.openReference(value,position)
+			return (
+				<View key={position}>
+					<TouchableOpacity onPress={() => {
+						this.props.openReference(value, position)
 					}}>
-					<View style={[styles.corner,{backgroundColor:'transparent'}]}>
-						<Text style={this.props.styles.textStyle}>{this.props.dataValue.bookName} {this.props.dataValue.chapterNumber}:{value}</Text>
-                       {/* {
+						<View style={[styles.corner, { backgroundColor: 'transparent' }]}>
+							<Text style={this.props.styles.textStyle}>{this.props.dataValue.bookName} {this.props.dataValue.chapterNumber}:{value}</Text>
+							{/* {
 						   this.props.dataValue.verses.length === 1 ? null :  <Icon name="clear" style={this.props.styles.iconReferClose}
```

Comment:
is this commented code needed?

Reply:
yes this may need for notes in next release if we will add delete verse or add new verse